exporter/views.py

```diff
@@ -92,9 +100,10 @@ class ExportCampaignToBagit(TemplateView):
     @method_decorator(login_required)
     def get(self, request, *args, **kwargs):
         campaign_slug = self.kwargs["campaign_slug"]
-        assets = get_latest_transcription_data(campaign_slug)
+        assets = get_latest_transcription_data(campaign_slug, True)
```

Comment:
I would really want this to name the keyword argument for clarity. I now prefer using the newer Python 3 syntax for the function declaration (`def foo(bar, *, baaz):`) which makes it a syntax error to pass a keyword argument by position.